### PR TITLE
Colombo

### PR DESCRIPTION
**Pull request by innovate-invent**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/colombo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  
+requirements:
+  run:
+    - openjdk
+
+test:
+  commands:
+    - SigiHMM --help
+    - mSigiHMM --help
+    - SigiCRF --help
```
Comment (bgruening) [inline recipes/colombo/meta.yaml]: You are not testing Colombo here. Would be nice to test this as well.

Comment (innovate-invent) [reply recipes/colombo/meta.yaml]: Colombo provides no CLI and requires a X11 display and user input to test.

**Comment by innovate-invent** [comment]
Colombo has hard coded relative paths that require the working directory to be its install path. The bash script just changes the working directory to its install folder. I don't think I will be able to modify it to conform with the java recommendations.

**Comment by bgruening** [comment]
@innovate-invent thanks!

**Pull request merged**